src/styles/blocks/catalog.scss

```
@@ -17,9 +17,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms, 300ms;
+    transition-property: color, transform;
 
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    &:hover {
+      color: #34568b;
```

don't forget to create variables for all colors

```suggestion
      color: $c-color;
```